.github/CODEOWNERS

```diff
@@ -391,7 +391,8 @@
 /app.yaml @vojtechjelinek
 /core/templates/dev/head/pages/Base.ts @vojtechjelinek
 /core/templates/dev/head/pages/base.html @vojtechjelinek
-/core/templates/dev/head/pages/footer.html @vojtechjelinek
+/core/templates/dev/head/pages/oppia_footer_directive.html @vojtechjelinek
+/core/templates/dev/head/pages/OppiaFooterDirective.ts @vojtechjelinek
```

Comment:
I am not sure whom to assign `OppiaFooterDirective.ts`. @seanlip @DubeySandeep Do you have any ideas?

Reply:
I think what you have here is fine.